src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ComboBoxTests.cs

```diff
@@ -617,6 +617,121 @@ public void ComboBox_DataSource_SetWithHandler_CallsDataSourceChanged()
         Assert.Equal(0, displayMemberCallCount);
     }
 
+    public class CustomComboBox : ComboBox
+    {
+        // Make methods private to improve encapsulation
+        private void RaiseOnDrawItem(DrawItemEventArgs e)
+        {
+            base.OnDrawItem(e);
+        }
```

Comment:
Let's go ahead and make this a body expression
```suggestion
        private void RaiseOnDrawItem(DrawItemEventArgs e) => base.OnDrawItem(e);
```